tests/unit/parser.js

```diff
@@ -3795,20 +3795,45 @@ exports["no let not directly within a block"] = function (test) {
   ];
 
   TestRun(test)
-    .addError(1, "Let declaration not directly within block.")
-    .addError(4, "Let declaration not directly within block.")
-    .addError(7, "Let declaration not directly within block.")
-    .addError(8, "Let declaration not directly within block.")
-    .addError(9, "Let declaration not directly within block.")
-    .addError(10, "Let declaration not directly within block.")
-    .addError(11, "Let declaration not directly within block.")
-    .addError(11, "Let declaration not directly within block.")
-    .addError(11, "Let declaration not directly within block.")
+    .addError(1, "Block scoped variable declaration not directly within block.")
```

Comment:
Lets do this better --- Make the "let" or "const" mode a variable in the message, and pick the right one to display. Makes it more readable